manim/mobject/vector_field.py

```diff
@@ -720,20 +748,29 @@ def outside_box(p):
             step = max_steps
             if not step:
                 continue
-            line = VMobject()
+            line = Line()
```

Comment:
A `Line` probably initializes some un-needed attributes, so a `VMobject` might be more appropriate. To get around incompatibility, you could use a `VGroup`, or add a check for the OpenGL renderer.

Reply:
Done, took the latter approach, as I think it is clearer than using a group